src/device-registry/stage-docker-build.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
+echo -e "\n ==== 🛠  🔧 🔩 Building API docker image 🔩  🔧  🛠  ==== \n"
```

Comment:
cc @danielogen

Reply:
Thanks, still trying to appreciate the benefit of having the build these images at the same time? 

- Isn't this introducing some tight coupling of the Kafka and device registry image creation?
- Cant some one create a new image on Kafka without changing the rest of the device registry logic

Reply:
@Baalmart the kafka connection is part of the device registry microservice just like the way the API endpoints are part of the microservice. Changing the underlying logic of the microservice requires building both images.